### PR TITLE
(#381)Translation-Of-BlogPost

### PR DESCRIPTION
**Pull request by ahmadakbaryNIH**
closes # 381

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = defineConfig({
     admin_username: 'admin',
     admin_password: '',
     test_site_section: '/about-cancer/understanding',
-    front_end_base_url: 'https://www-dev-ac.cancer.gov'
+    front_end_base_url: 'https://www-test-ac.cancer.gov'
```
Comment (KateMashkinaNIH) [inline cypress.config.js:17]: this should be reverted

File: cypress/e2e/CMS/$BlogPosts/index.js [hunk 2]
```diff
@@ -250,4 +291,89 @@ Then('the promo image is matching the earlier selected image', () => {
...
+    const day = date.getDate();
+    const expectedDate = `${day} de ${month} de ${currYear}`;
+    cy.get('p>time').should('include.text', expectedDate);
+})
+
+And('user clicks on title with url spanish path {string} site section plus {string}', (spPath, purl) => {
+    cy.get(`a[href='${spPath}/${purl}-${randomNum}']`).click();
+})
+
+Given('user is navigating to the front end site with spanish path {string} site section plus {string}', (spPath, title) => {
```
Comment (KateMashkinaNIH) [inline cypress/e2e/CMS/$BlogPosts/index.js:376]: so this step is saying site section, but in reality you do not need it 
can you rephrase step in a feature file and update this one to say 
user is navigating to the front end site with spanish path {string} plus {string}

**Review by KateMashkinaNIH — CHANGES_REQUESTED** [review]

**Comment by ahmadakbaryNIH** [comment]
image
